examples/foo-missionmodel/src/main/java/gov/nasa/jpl/aerie/foomissionmodel/activities/ZeroDurationUncontrollableActivity.java

```diff
@@ -0,0 +1,14 @@
+package gov.nasa.jpl.aerie.foomissionmodel.activities;
+
+import gov.nasa.jpl.aerie.foomissionmodel.Mission;
+import gov.nasa.jpl.aerie.merlin.framework.annotations.ActivityType;
+import gov.nasa.jpl.aerie.merlin.framework.annotations.ActivityType.EffectModel;
+
+import static gov.nasa.jpl.aerie.merlin.framework.ModelActions.delay;
+
+@ActivityType("ZeroDurationUncontrollableActivity")
+public final class ZeroDurationUncontrollableActivity {
+  @EffectModel
+  public void run(final Mission mission) {
```

Comment:
By not including a duration, does that make the duration both uncontrollable and set it to zero, implicitly?